tests/newlines.html

```diff
@@ -3,11 +3,7 @@
 <div>
 Hello
 <br>
-</div>
```

Comment:
I'm concerned that you had to change existing behaviour for this change, as this package is used by many thousands of people, I would like the existing behaviour to continue (as long as it's reasonable). Can you revert the changes to the existing test cases and instead add new test cases illustrating the new behaviour?

Alternatively, can you provide screenshots showing that the original test was incorrect c.f. rendering HTML in a browser?